apps/frontend/src/utils/meet.ts

```diff
@@ -0,0 +1,37 @@
+import { startOfWeek, endOfWeek, isWithinInterval, format } from "date-fns"
+import { Meet } from "../lib/@types/Items/Meet"
+
+export function getCurrentWeekMeets(meets: Meet[] = []): Meet[] {
+  const weekStart = startOfWeek(new Date())
+  const weekEnd = endOfWeek(new Date())
+
+  return meets.filter((meet) =>
+    isWithinInterval(new Date(meet.metadata.start.dateTime), {
+      start: weekStart,
+      end: weekEnd,
+    })
+  )
+}
+
+export function formatMeetDate(date: string): string {
+  return format(new Date(date), "EEE, MMM dd")
+}
+
```

Comment:
Can be generic function which can be used for other things too not just for meets like
formatDate, formatTimeRange etc